src/ActionModal/components/SendModal.vue

```diff
@@ -256,7 +256,13 @@ export default {
       this.sending = false
     },
     setMaxAmount() {
-      this.amount = this.selectedBalance.amount
+      if (this.network.startsWith(`terra`)) {
+        const terraTax = 0.008
+        this.amount =
```

Comment:
ignores fees but we are ignoring that everywhere?

Reply:
? what do you mean?

Reply:
this sets the max to available balance - tax. but the actual max is available balance - tax - fees